README.md

```diff
@@ -67,6 +68,10 @@ A `stable` tag can be trusted that it is the latest version in which there are n
 * It has been out for at least 2 weeks or is a CVE patch with no Fluent Bit changes
 * No bugs have been reported in Fluent Bit which we expect will have high impact for AWS customers. This means bugs in the components that are most frequently used by AWS customers, such as the AWS outputs of the tail input
 
+#### Using the init tag
+
+A `init` tag indicates that this is an image contains init process and supports multi-config. Init tag is used in addition to our other tags, e.g. `aws-for-fluent-bit:init-latest` means this is a latest released image supports multi-config. For more information about the usage of multi-config please see our [use case guide](https://github.com/aws/aws-for-fluent-bit/blob/mainline/use_cases/init-process-for-fluent-bit/README.md) and [FireLens example](https://github.com/aws-samples/amazon-ecs-firelens-examples/tree/mainline/examples/fluent-bit/multi-config-support).
```

Comment:
nit: should be "An init tag" since init starts with a vowel.. though I think "The init tags indicate that an image contain.." is the way I'd probably write it

Reply:
nit: Also I am not sure what "Init tag is used in addition to our other tags" means.. I think you mean the init tag is different than the other tags? May be skip this and just go straight to "For example, `aws-for-fluent-bit:init-latest` means..."